frontend-react/src/styles/_components.scss

```diff
@@ -9,6 +11,43 @@ hr {
     padding: 0.25rem 0.25rem;
 }
 
+// MessageDetails.tsx
+.usa-table thead th {
```

Comment:
Hmmm can you wrap all of these rules in a Message Tracker-specific modifier class?  We don't want these rules to leak out and apply globally to other tables -- at least not until we're using one unified table component.

I realize there's some `message-receivers`-specific classes below, but I think it'd be safer to wrap everything and only unwrap when we're sure that these are styles we want to apply globally.

Reply:
Yeah! Good call, that should be standard practice until we use CSS modules.